ARM.CMSIS-Compiler.pdsc

```diff
@@ -482,4 +482,21 @@
       </files>
     </component>
   </components>
+
+  <examples>
+    <example name="retarget" folder="example" doc="README.md">
+      <description>Simple example retargeting stdout to USART.</description>
+      <board name="V2M-MPS2" vendor="ARM"/>
+      <project>
+        <environment name="cmsis" load="retarget.csolution.yml"/>
+      </project>
+      <attributes>
```

Comment:
I suggest to remove <attributes> from the examples.

Reply:
You are talking about removing the `<attributes>`?